README.md

```diff
@@ -63,6 +63,24 @@ The following Visual Studio Code settings are available for the Arduino extensio
 - `arduino.additionalUrls` - Additional URLs for 3rd party packages. You can have multiple URLs in one string with comma(,) as separator, or have a string array. The default value is empty.
 - `arduino.logLevel` - CLI output log level. Could be info or verbose. The default value is `"info"`.
 
+The following settings are per sketch settings of the Arduino extension. You can find them in
+`.vscode/arduino.json` under the workspace.
+
+```json
+{
+    "sketch": "example.ino",
+    "port": "COM5",
+    "board": "adafruit:samd:adafruit_feather_m0",
+    "output": "../build",
+    "debugger": "jlink"
+}
+```
+- `sketch` - The main sketch file name of Arduino.
```

Comment:
For all the settings here, tell user the default value if they have. Users might use this doc as a reference and change the value to default if something might not work in their dev env.